packages/react-composites/tests/app/callwithchat/index.tsx

```diff
@@ -8,8 +8,9 @@ import { HermeticApp, shouldLoadHermeticApp } from './HermeticApp';
 import { LiveApp } from './LiveApp';
 import { parseQueryArgs } from './QueryArgs';
 
-const queryArgs = parseQueryArgs();
 initializeIconsForUITests();
+const queryArgs = parseQueryArgs();
+console.log(`Parsed args for test app: ${JSON.stringify(queryArgs)}`);
```

Comment:
Are we logging anything we shouldn't here? OK if we log a token?

Reply:
Thanks for catching this. No I don't think we should log a token. Will remove.